prometheus/config.go

```diff
@@ -33,10 +33,12 @@ func WriteConfig(configPath string, scrapes map[string]Scrape,
 		c.InsertScrapesFromDir(configsDir)
 	}
 
+	c.RuleFiles = []string{"/run/secrets/*.rules"}
```

Comment:
How do you feel about using: https://godoc.org/github.com/spf13/afero#Glob to explicitly scan for all the rules, adding these rules to `prometheus.yml` and logging the rules here?

Reply:
Hi @thomasjpfan,

I am using secrets because since the beginning I am injecting the scrapes via secret, so this was my first approach.

I believe I don't need to use https://godoc.org/github.com/spf13/afero#Glob because Prometheus already implement it.

Yes @vfarcic , I can provide some documentation and release notes.

Reply:
@albertogviana Yes. Please add the docs and release notes.

@thomasjpfan Do you think we can merge this after the docs and RNs are done by @albertogviana ?